src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -289,6 +294,32 @@ def _send_verification_code_to_user(
     )
 
 
+def _send_privacy_request_receipt_email_to_user(
+    db: Session, policy: Policy, email: Optional[str]
+) -> None:
+    """Helper function to send request receipt email to the user"""
+    if not email:
+        logger.error(
+            IdentityNotFoundException(
+                "Identity email was not found, so request receipt email could not be sent."
+            )
+        )
+    request_types: Set[str] = set()
+    for action_type in ActionType:
+        if policy.get_rules_for_action(action_type=ActionType(action_type)):
+            request_types.add(action_type)
+    try:
+        dispatch_email(
+            db=db,
+            action_type=EmailActionType.PRIVACY_REQUEST_RECEIPT,
+            to_email=email,
+            email_body_params=RequestReceiptBodyParams(request_types=request_types),
+        )
+    except EmailDispatchException as e:
+        # catch early since this failure isn't fatal to privacy request, unlike the subject id verification email
+        logger.error(e)
```

Comment:
One of the EmailDispatchExceptions can log the exception error plain, is that a concern here?

Reply:
to be safe, I can mask for potential pii here